ddtrace.gemspec

```diff
@@ -6,8 +6,8 @@ require 'ddtrace/version'
 
 Gem::Specification.new do |spec|
   spec.name                  = 'ddtrace'
-  spec.version               = Datadog::VERSION::STRING
-  spec.required_ruby_version = [">= #{Datadog::VERSION::MINIMUM_RUBY_VERSION}", "< #{Datadog::VERSION::MAXIMUM_RUBY_VERSION}"]
+  spec.version               = DDTrace::VERSION::STRING
```

Comment:
I see this making sense but I have an awkward feeling introducing a `DDTrace` constant with all the effort to move stuff towards `Datadog`.

Reply:
I think `DDTrace` looks strange, but it's very descriptive and accurate: it represents information tied to the gem itself, not to Datadog products.

Reply:
(I'm sure open to suggestions, but I'm not against the current naming)

Reply:
Yup that's exactly how I feel.

> it represents information tied to the gem itself, not to Datadog products.

What about `Datadog::Gem`? `Datadog::Gem::VERSION` has a nice ring to it. I'm thinking how in the future we plan to transition from `ddtrace` to `datadog`.

Reply:
My strategic reason for doing this is that it allows us to introduce a `datadog` gem later without having a conflict with this one (Which will likely contain `Datadog::VERSION`.)

Reply:
That's a quite reasonable rationale 👍